docs/manage/reference/accounts.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: "Account Management"
+title: "Account Management"
+weight: 1000
+layout: "docs"
+type: "docs"
+no_list: true
+description: "Log in and out of your Viam account."
+aliases:
+  - /fleet/account/
+  - /cloud/account/
+---
+
+The [Viam app](https://app.viam.com/) is a web UI for managing and building machines.
```

Comment:
I guess this is here for people who are being linked here from a link that says "Create a Viam app account and log in" or similar, so they know what they're signing up for. But, this sentence alone feels like it's not directly tied to the page contents. For example (just one idea off hand):
```suggestion
The [Viam app](https://app.viam.com/) is a web UI for managing and building machines.
To use the app, each user needs a free account.
Accounts can be [assigned access](/manage/manage/access/) to different groups of machines.
```